packages/admin/cms-admin/src/contentScope/Select.tsx

```diff
@@ -13,24 +15,56 @@ export interface ContentScopeSelectProps {
     defaultLabel?: string;
     icon?: (p: SvgIconProps) => JSX.Element;
     disabled?: boolean;
+    searchable?: boolean;
 }
 
-export default function ContentScopeSelect({ value, onChange, values, defaultLabel = "", icon, disabled }: ContentScopeSelectProps): JSX.Element {
+export default function ContentScopeSelect({
+    value,
+    onChange,
+    values,
+    defaultLabel = "",
+    icon,
+    disabled,
+    searchable = false,
+}: ContentScopeSelectProps): JSX.Element {
+    const intl = useIntl();
+
     const Icon = icon || null;
     // @TODO: styling for disabled-state
+
+    const [searchValue, setSearchValue] = React.useState<string>("");
+
+    const filteredValues = searchable ? values.filter((item) => item.value.toLowerCase().includes(searchValue.toLowerCase())) : values;
+
     return (
         <AppHeaderDropdown buttonChildren={value ? value.label || value.value.toUpperCase() : defaultLabel} startIcon={Icon ? <Icon /> : undefined}>
             {(hideDropdown) => (
                 <List>
-                    {values.map(({ value: v, label: l }) => (
-                        <ListItem
+                    {searchable && (
+                        <InputBase
+                            sx={{ m: 1 }}
+                            startAdornment={<Search />}
```

Comment:
Please wrap icons used as input-adornments with `<InputAdornment />` like in the MUI examples: https://mui.com/material-ui/react-text-field/#icons

```suggestion
                            startAdornment={<InputAdornment position="start"><Search /></InputAdornment>}
```